iam-persistence/src/main/resources/db/migration/mysql/V100__alter_saved_user_auth_info_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE saved_user_auth_info MODIFY info_val varchar(256) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
case-insensitive?

Reply:
yes, I left it case-insensitive as before. Isn't it good?

Reply:
It depends on what that field represents